src/components/UI/Card/CardContainer.styled.js

```diff
@@ -1,11 +1,11 @@
 import styled from 'styled-components';
 
 const CardContainer = styled.li`
-	background-color: #ffa375;
-	padding: 20px;
-	margin: 20px;
+	background-color: var(--orange);
+	padding: 15px;
+	margin-top: 15px;
 	border-radius: 20px;
-	width: 33rem;
+	width: 360px;
```

Comment:
I'd probably use percentages rather than fixed widths, as well as potentially a couple of breakpoints. Ultimately it's not something you need to concern yourself with at this point, but something to keep in mind for future reference.

Reply:
Thank you, good to know! One last question: What do you mean by breakpoints?

Reply:
Thank you! :)